index.js

```diff
@@ -24,12 +24,14 @@ module.exports = function(source) {
   }
 
   var svgo = new Svgo(config);
-  svgo.optimize(source, function(result) {
+  svgo.optimize(source)
+  .then(function(result) {
     if (result.error) {
```

Comment:
Do they still pass error like this after switching to promises? Also we should handle the failure case of the promise.